util/dataset.go

```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/elastic/go-ucfg/yaml"
 )
 
+const (
+	DirIngestPipeline = "ingest-pipeline"
```

Comment:
@mtojek I'm tempted to say we should rename this to `ingest_pipeline` but don't have a strong argument for either. Any opinion?

Reply:
I'm fine with this path.

Reply:
Which one, `-` or `_`?

Reply:
I mean, I'm with both, so also with the underscore :)

Reply:
I was hoping you would have a preference to tip it in one direction :-)